kerl

```diff
@@ -2052,9 +2067,9 @@ list_add() {
                 return 0
             fi
         done <"$KERL_BASE_DIR/otp_$1"
-        echo "$2" >>"$KERL_BASE_DIR/otp_$1" || return 1
+        echo "$2" >>"$KERL_BASE_DIR/otp_$1"
     else
-        echo "$2" >"$KERL_BASE_DIR/otp_$1" || return 1
+        echo "$2" >"$KERL_BASE_DIR/otp_$1"
```

Comment:
I searched the Git history for this and there were relevant changes in https://github.com/kerl/kerl/commit/ca96e227e1dc55c3a4f7b930e6d766f048efb2b8. I'll re-apply them to the updated code.

Reply:
And since I added calls to `tee` to create the `otp_releases` I'm also gonna check and error out on those, if required.

Reply:
(this change was later reversed)